app/gui2/src/components/ComponentBrowser.vue

```diff
@@ -19,52 +37,301 @@ function positionAtMouse(): boolean {
   return true
 }
 
-const transform = computed(() => {
-  const nav = props.navigator
-  const pos = scenePosition.value
-  return `${nav.transform} translate(${pos.x}px, ${pos.y}px) scale(${
-    1 / nav.scale
-  }) translateY(-100%)`
+// === Components List and Positions ===
+
+const componentStore = useComponentsStore()
+
+const visibleComponents = computed(() => {
+  if (scroller.value) {
```

Comment:
I don't like early returns in _any_ language, because:
1. In short function, nesting usually does not hurt as much.
2. In long function, early return may _hurt_ a lot.